.github/workflows/node.js.yml

```diff
@@ -0,0 +1,52 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: GH Pages deploy
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Install
+        run: npm ci
+      - name: Build
+        run: npm run build --if-present
+      - name: Test
+        run: npm test
+      - name: Upload Artifact
+        uses: actions/upload-artifact@master
+        with:
+          name: J40Static
+          path: public
+  deploy:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - name: Checkout source
+        uses: actions/checkout@v2
+      - name: Download Artifacts
+        uses: actions/download-artifact@master
+        with:
+          name: J40Static
+          path: public
+      - name: Deploy 🚀
```

Comment:
Is unicode safe for the name? I guess it is if it works.

Reply:
Good callout - It does seem acceptable to have them - this was taken from a tutorial but I've seen emoji in a few other projects also.